### PR TITLE
LPS-153200 Update contains function to accept numbers

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java [hunk 1]
```diff
@@ -48,11 +48,7 @@ public Boolean apply(Object object1, Object object2) {
 			return apply(object1.toString(), object2);
 		}
 
-		if ((object1 instanceof String) && (object2 instanceof String)) {
-			return apply((String)object1, (String)object2);
-		}
-
-		return false;
+		return apply(object1.toString(), object2.toString());
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java:51]: @seltong could you check if `object2` is null (same as `object1` [here](https://github.com/liferay-forms/liferay-portal/blob/31b48ccdc8ec3c4eb5571765b7246f0eebf39555/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java#L37))? Thanks!

**Comment by carolmariaabb** [comment]
ci:test:forms

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:forms

**Comment by seltong** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**